changelog.d/6140.bugfix

```diff
@@ -0,0 +1 @@
+Make the `synapse_port_db` script create the right indexes on a new PostgreSQL database.
```

Comment:
it's likely this is going to end up in a different release to #6102 : can you give it a different changelog?